orchestrator.py

```diff
@@ -0,0 +1,131 @@
+            message['response'] = self.ai_llm_service.run_llm(msgs)
+            self.handle_translation(message)
+        except Exception as e:
+            print(f"Exception in request_llm_response: {e}")
+
+    def handle_translation(self, message):
+        # Do this all as one piece, at least for now
+        llm_response = message['response']
+        out = ''
+        for chunk in llm_response:
+            if len(chunk["choices"]) == 0:
+                continue
+            if "content" in chunk["choices"][0]["delta"]:
+                if chunk["choices"][0]["delta"]["content"] != {}: #streaming a content chunk
+                    next_chunk = chunk["choices"][0]["delta"]["content"]
+                    out += next_chunk
+        #sentence = self.ai_tts_service.run_tts(out)
+        message['translation'] = out
+        message['translation_language'] = self.language
+        self.enqueue(TranslatorScene, message=message)
+
+    def request_tts(self, message):
+        try:
+            for chunk in self.ai_tts_service.run_tts(message):
+                yield chunk
+        except Exception as e:
+            print(f"Exception in request_tts: {e}")
+            
+    def handle_audio(self, audio):
+        b = bytearray()
+        final = False
+        try:
+            for chunk in audio:
+                b.extend(chunk)
+                l = len(b) - (len(b) % 640)
+                if l:
+                    self.microphone.write_frames(bytes(b[:l]))
+                    b = b[l:]
+        
+            final = True
+            self.microphone.write_frames(bytes(b))
+            time.sleep(len(b) / 1600)
+        except Exception as e:
+            print(f"Exception in handle_audio: {e}", len(b), final)
+
+    def enqueue(self, scene_type, **kwargs):
+        # Take the newly created scene object, call its prepare function, and queue it to perform
+        kwargs['orchestrator'] = self
+        self.scene_queue.put(scene_type(**kwargs))
+        pass
+
+    def action(self):
+        # start playing scenes. This allows us to prepare scenes
+        # before we actually start playback
+        self.playback_thread = Thread(target=self.playback)
+        self.playback_thread.start()
+
+    def playback(self):
+        while True:
+            try:
+                if self.stop_threads:
+                    print("🎬 Shutting down playback thread")
+                    break
+                scene = self.scene_queue.get(block=False)
+                if 'sentence' in scene.__dict__:
+                    print(f"🎬 Performing scene: {type(scene).__name__}, {scene.sentence}")
+                else:
+                    print(f"🎬 Performing sentenceless scene: {type(scene).__name__}")
+                scene.perform()
+            except Empty:
+                time.sleep(0.1)
+                continue
```

Comment:
I don't _think_ you need `continue` hear - it should continue already, because you didn't re-raise.